core-services/prow/02_config/openshift/cluster-samples-operator/_pluginconfig.yaml

```diff
@@ -1,3 +1,10 @@
+label:
+  restricted_labels:
+    openshift/cluster-samples-operator:
+    - allowed_users:
```

Comment:
I *think* you should have an entry like https://github.com/openshift/release/blob/master/core-services/prow/02_config/openshift/openshift-controller-manager/_pluginconfig.yaml#L4-L5 and include something similar in your change here

@adambkaplan @coreydaley would you guys agree?

wdyt @dperaza4dustbit ?

Reply:
yes, you need to allow the patch managers to add this label, too.

Reply:
OK, changing, thanks